sdk/java-sdk-protobuf/src/main/java/kalix/javasdk/action/Action.java

```diff
@@ -65,15 +95,34 @@ public final Effect.Builder effects() {
     return ActionEffectImpl.builder();
   }
 
-  /** Returns a {@link TimerScheduler} that can be used to schedule further in time. */
+  /**
+   * Returns a {@link TimerScheduler} that can be used to schedule further in time.
+   */
   public final TimerScheduler timers() {
     ActionContextImpl impl =
-        (ActionContextImpl)
-            actionContext("Timers can only be scheduled or cancelled when handling a message.");
+      (ActionContextImpl)
+        actionContext("Timers can only be scheduled or cancelled when handling a message.");
     return new TimerSchedulerImpl(impl.messageCodec(), impl.system());
   }
 
   /**
+   * An Effect is a description of what Kalix needs to do after the command is handled.
+   * You can think of it as a set of instructions you are passing to Kalix. Kalix will process the instructions on your
+   * behalf.
+   * <p>
+   * Each Kalix component defines its own effects, which are a set of predefined
+   * operations that match the capabilities of that component.
+   * <p>
+   * An Action Effect can either:
+   * <p>
+   * <ul>
+   *   <li>reply with a message to the caller
+   *   <li>reply with a message to be published to a topic (in case the method is a publisher)
+   *   <li>forward the message to another component
+   *   <li>return an error
+   *   <li>ignore the call
```

Comment:
I think this doesn't align with "what Kalix needs to do after the command is handled". If the command has been handled how can it be ignored after the fact?
How about this? Ignore further processing of the message, drop it, and process next element in the topic or return NotFound for a Request/Reponse. 

I would add that and effect is the last thing an action can do and must do for when processing a message.

Reply:
I prefer to not be too verbose on this description because we have more information down on the method itself. 

For example, the `ignore` method is explaining in detail what are the semantics.